doc/user/bgp.rst

```diff
@@ -692,6 +692,7 @@ Route Aggregation
 .. clicmd:: aggregate-address A.B.C.D/M
 
    This command specifies an aggregate address.
+   This command is applicable for IPV4 and IPV6 address-family.
```

Comment:
Please actually update the documentation to indicate both v4 and v6 addresses on lines 691-692, 697-698, 703-704, 708-709

Reply:
I have added the example configuration. Please have a look.

Reply:
Please update the documentation as I have outlined.  Having an example block is fine though

Reply:
I have added the new index directive for IPV6 address family.
Please check it once.